pkg/jsonschema/json-schema.go

```diff
@@ -92,8 +86,21 @@ func createNode(name string, v model.TranslatedVariable, options CreateSchemaOpt
 	if v.Variable.Validation != nil && v.ConditionAsString != nil {
 		err = parseConditionToNode(v.Variable.Validation.Condition, *v.ConditionAsString, name, &node)
 		// if an error occurs, log it and continue.
-		if err != nil {
-			fmt.Printf("couldn't apply validation for %q with condition %q. Error: %v\n", name, *v.ConditionAsString, err)
+		if err != nil && !options.SuppressLogging {
```

Comment:
Having logging configuration affect control flow will work, but there may be a neater way to do the same thing.

Reply:
Probably, but slog gives me a headache and this does the job for now